install_config/syncing_groups_with_ldap.adoc

```diff
@@ -433,6 +437,156 @@ users:
+
+----
+Error determining LDAP group membership for "<group>": membership lookup for user "<user>" in group "<group>" failed because of "search for entry with dn="<user-dn>" would search outside of the base dn specified (dn="<base-dn>")".
+----
+
+This often indicates a mis-configured `baseDN` in the `usersQuery`. However,
+in cases where the `baseDN` intentionally does not contain some of the members
+of the group, setting `tolerateMemberOutOfScopeErrors: true` will allow the group
+sync to continue. Out of scope members will be ignored.
+
+Similarly, when the group sync process fails to locate a member for a group, it fails
+outright with errors like:
+
+----
+Error determining LDAP group membership for "<group>": membership lookup for user "<user>" in group "<group>" failed because of "search for entry with base dn="<user-dn>" refers to a non-existent entry".
+
+Error determining LDAP group membership for "<group>": membership lookup for user "<user>" in group "<group>" failed because of "search for entry with base dn="<user-dn>" and filter "<filter>" did not return any results".
+----
+
+This often indicates a mis-configured `usersQuery`. However, in cases where the
+group contains member entries that are known to be missing, setting 
+`tolerateMemberNotFoundErrors: true` will allow the group sync to continue. Problematic
+members will be ignored.
+
+[WARNING]
+====
+Enabling error tolerances for the LDAP group sync will cause the sync process to ignore
+problematic member entries. If the LDAP group sync is not configured correctly, this could
+result in synced OpenShift groups missing members.
+====
+
+.LDAP Entries Using RFC 2307 Schema With Problematic Group Membership: *_rfc2307_problematic_users.ldif_*
+====
+[source,ldif]
+----
+  dn: ou=users,dc=example,dc=com
+  objectClass: organizationalUnit
+  ou: users
+
+  dn: cn=Jane,ou=users,dc=example,dc=com
+  objectClass: person
+  objectClass: organizationalPerson
+  objectClass: inetOrgPerson
+  cn: Jane
+  sn: Smith
+  displayName: Jane Smith
+  mail: jane.smith@example.com
+
+  dn: cn=Jim,ou=users,dc=example,dc=com
+  objectClass: person
+  objectClass: organizationalPerson
+  objectClass: inetOrgPerson
+  cn: Jim
+  sn: Adams
+  displayName: Jim Adams
+  mail: jim.adams@example.com
+
+  dn: ou=groups,dc=example,dc=com
+  objectClass: organizationalUnit
+  ou: groups
+
+  dn: cn=admins,ou=groups,dc=example,dc=com
+  objectClass: groupOfNames
+  cn: admins
+  owner: cn=admin,dc=example,dc=com
+  description: System Administrators
+  member: cn=Jane,ou=users,dc=example,dc=com
+  member: cn=Jim,ou=users,dc=example,dc=com
+  member: cn=INVALID,ou=users,dc=example,dc=com <1>
+  member: cn=Jim,ou=OUTOFSCOPE,dc=example,dc=com <2>
+----
+<1> This member doesn't exist on the LDAP server.
+<2> This member may exist, but is not under the base DN in the
+user query for the sync job.
+====
+
+In order to tolerate errors, the following additions to your sync configuration
```

Comment:
Maybe:

s/In order to tolerate errors,/In order to tolerate the errors in the above example,/